### ISSUE TITLE
ckpt reproduce

### ISSUE THREAD

**Issue by junho-jo** [zhang-tao-whu/DVIS#2]
Hi, Author!

When i evaluated the performance using the trained model you provided, [DVIS_online_r50](https://github.com/zhang-tao-whu/DVIS/blob/main/configs/youtubevis_2019/DVIS_Online_R50.yaml) and [DVIS_offline_r50](https://github.com/zhang-tao-whu/DVIS/blob/main/configs/youtubevis_2019/DVIS_Offline_R50.yaml), the performance was similar with paper.

To reproduce the ckpt by re-training, i followed your configuration. 

```
# train the DVIS_Online
python train_net_video.py \
  --num-gpus 8 \
  --config-file ./configs/youtubevis_2019/DVIS_Online_R50.yaml \
  MODEL.WEIGHTS ./ckpt/minvis_ytvis19_R50.pth
```

[benchmark score of reproduced-online method ](https://github.com/zhang-tao-whu/DVIS/files/11812006/stdout.txt) was simliar with paper.

```
# train the DVIS_Offline
python train_net_video.py \
  --num-gpus 8 \
  --config-file ./configs/youtubevis_2019/DVIS_Offline_R50.yaml \
  MODEL.WEIGHTS ./ckpt/DVIS_online_ytvis19_r50.pth \ 
```

However, when I tried learning the offline method according to the code provided above, the reproduced performance was significantly lower ([ytvis2019: 1 AP](https://github.com/zhang-tao-whu/DVIS/files/11812168/offilne-retraining.txt)). Could you please check the training code or configuration?
(never change anything in your code)

**Comment by zhang-tao-whu** [comment]
OK. I will retrain DVIS-offline tonight and try to identify the bug. If you could provide me with your training log, that would be great.

**Comment by junho-jo** [comment]
This is my training [log.txt](https://github.com/zhang-tao-whu/DVIS/files/11862457/log.txt)! Please let me know if there is anything strange or something I need to check.

**Comment by zhang-tao-whu** [comment]
Hi! I tried to retrain DVIS-offline using 4 2080ti GPUs and set the batch size to 4 (ensuring one batch per GPU), and it worked well. Here are the training logs  
[nohup.txt](https://github.com/zhang-tao-whu/DVIS/files/11862574/nohup.txt)
 and results 
[stdout - 2023-06-26T102917.565.txt](https://github.com/zhang-tao-whu/DVIS/files/11862576/stdout.-.2023-06-26T102917.565.txt)
. 

I found that your training log is not normal. After the training is completed, the total_loss should be reduced to around 10. If you haven't changed the configuration file (ensuring that the GPU numbers and batch numbers are same), it may be due to the version of PyTorch. Previously, I also encountered this phenomenon using PyTorch 1.11 on 8 V100 GPUs. If possible, I suggest that you create a new virtual environment and configure a different version of PyTorch (a earlier or newer version). I have run well on PyTorch 1.9.0 (cuda 10.2 or 11.3).

**Comment by junho-jo** [comment]
As you mentioned, the settings as written in [install.md](https://github.com/zhang-tao-whu/DVIS/blob/main/INSTALL.md) work just fine (torch==1.9.0, cuda==11.1) . 
I appreciate for your kind reply!! thx!

**Comment by Kunaldargan** [comment]
@zhang-tao-whu ,
Thank you for providing the code and installation instruction.
I'm using  following version of pytorch with cuda 11.1


>  Name                    Version                   Build  Channel
>  torch                     1.9.0+cu111              pypi_0    pypi

while training is completed on a dataset similar to youtube-vis 2019. 
In offline training final loss is around 25, In online training final loss is around 5.
During evaluation: mAP is 0.01 for Offline setting and 0.22 for Online setting.

Please review.

**Comment by zhang-tao-whu** [comment]
Since the segmenter is frozen during the training process of DVIS (tracker and refiner), it is necessary to fine-tune the segmenter using your own dataset.

Please refer to [GETTING_STARTED.md](https://github.com/zhang-tao-whu/DVIS/blob/main/GETTING_STARTED.md). In the 'Training on a new dataset' section of this document, I provide a detailed explanation of how to train DVIS on a completely new dataset. Please follow this process and try again.
